src/main/java/org/webshar/hrms/controllers/RoleController.java

```diff
@@ -0,0 +1,28 @@
+package org.webshar.hrms.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.webshar.hrms.dao.exceptions.EntityNotFoundException;
+import org.webshar.hrms.response.Response;
+import org.webshar.hrms.services.impl.RoleServiceImpl;
+
+@RestController
+@RequestMapping("/")
+public class RoleController
+{
+  @Autowired
+  RoleServiceImpl roleService;
+
+
+  @RequestMapping(value = "/services/api/web/hrms/roles/{role_id}", method = RequestMethod.GET)
+  @ResponseBody
```

Comment:
As we are using `RestController` we don't need `ResponseBody` annotation here.
`RestController` is equal to `Controller + ResponseBody`